storm-client/src/jvm/org/apache/storm/utils/Utils.java

```diff
@@ -109,6 +109,8 @@
 
 import javax.security.auth.Subject;
 
+import static org.apache.commons.lang.exception.ExceptionUtils.*;
```

Comment:
nit: don't use *

Reply:
+1. A little odd that checkstyle didn't catch this, since it's in the rules https://github.com/apache/storm/blob/master/storm-checkstyle/src/main/resources/storm/storm_checkstyle.xml#L82. @kishorvpatil could you check if maybe the number of allowed style violations for storm-client can be reduced?

Reply:
Updated.